config/flask_config.py

```diff
@@ -64,6 +64,7 @@
 
 # basic flask settings
 config['PORT'] = int(config['PORT'])
+config['DEBUG'] = (config['DEBUG'] == 'TRUE')
```

Comment:
So we just moved to a new configuration system, and we forgot to include the debug setting.  This sets the flask debug mode on if the debug environment variable is `"TRUE"`.  The way the environment variables work is that they can only be set to strings, so into have to be cast to into (see the above line) or books have to be created from strings (this line)

Does that make sense?  Could we make this more clear?